### PR TITLE
Add and improve translator comments and tooltips for peers tab address fields

### PR DESCRIPTION
**Pull request by jonatack**
Per translator feedback in this thread: https://github.com/bitcoin-core/gui/pull/526#discussion_r809237830

*"The lack of string context in Transifex is a real problem for this project, as proper context (dev notes and/or screenshots) are essential to achieve quality translations."*

This pull adds developer notes for transifex translators via `extracomment` tags, and it improves the existing ones and their tooltips with more context, clarity and completeness for the following peer tab fields as a follow-up to bitcoin-core/gui#526:

- address relay
- addresses processed
- addressed rate-limited

It looks like only six lines of diff, but they are loooong lines.

If this is the right direction, the same can be done for other fields in follow-ups.

### CODE REVIEW

File: src/qt/forms/debugwindow.ui [hunk 1]
```diff
@@ -1594,10 +1594,10 @@
                <item row="23" column="0">
                 <widget class="QLabel" name="peerAddrRelayEnabledLabel">
                  <property name="toolTip">
-                  <string extracomment="Tooltip text for the Address Relay field in the peer details area.">Whether we relay addresses to this peer.</string>
+                  <string extracomment="Tooltip text for the Address Relay field in the peer details area, which displays whether we relay addresses to this peer (Yes/No).">Whether we relay addresses to this peer (Yes/No).</string>
```
Comment (promag) [inline src/qt/forms/debugwindow.ui:1597]: Why the 2nd Yes/No?

**Review by promag — COMMENTED** [review]
Concept ACK, seems to be in line with what was suggested/requested.

File: src/qt/forms/debugwindow.ui (hunk 1 above)
Comment (jonatack) [reply src/qt/forms/debugwindow.ui:1597]: Which one would you remove?

Comment (promag) [reply src/qt/forms/debugwindow.ui:1597]: The second, leave the string unchanged.

Comment (jonatack) [reply src/qt/forms/debugwindow.ui]: Thanks, done.

**Comment by jonatack** [comment]
Updated per @promag feedback (thanks!)

**Review by jarolrod — APPROVED** [review]
ACK bcd72ef18f14d7a457db6c8b835a571dec1f2beb

I've been collecting thoughts on how to formulate translator comments [here](https://github.com/bitcoin-core/gui-qml/blob/main/src/qml/doc/translator-comments.md) and the translator comments introduced here tick the boxes. 

in reference to the translator comment documentation I've prepared: in 1df9daaa457a57554ba2b334f989bf5d599177bf you allude to what the value of the field is, I wonder if I should include that point as required context in the doc.

If you retouch, and definitely not necessary, you could rename the PR to `Add and improve peers tab address translator comments and tooltips` and similarly update the commits to state they are updating translator comments instead of extracomment.

**Comment by jonatack** [comment]
> ACK [bcd72ef](https://github.com/bitcoin-core/gui/commit/bcd72ef18f14d7a457db6c8b835a571dec1f2beb)
> 
> in reference to the translator comment documentation I've prepared: in [1df9daa](https://github.com/bitcoin-core/gui/commit/1df9daaa457a57554ba2b334f989bf5d599177bf) you allude to what the value of the field is, I wonder if I should include that point as required context in the doc.

Describing the value of the field where relevant and helpful seems like a good idea.

> If you retouch, and definitely not necessary, you could rename the PR to `Add and improve peers tab address translator comments and tooltips` and similarly update the commits to state they are updating translator comments instead of extracomment.

Done!

**Review by jarolrod — COMMENTED** [review]
re-ACK [4d2b503](https://github.com/bitcoin-core/gui/commit/4d2b503d6cbb593cf75cf053fd480de64b3f4fc0)

Thanks for picking up the suggestions!

**Review by hebasto — APPROVED** [review]
ACK 4d2b503d6cbb593cf75cf053fd480de64b3f4fc0, I have reviewed the code and it looks OK, I agree it can be merged.

**Pull request merged**